packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.js

```diff
@@ -32,7 +32,9 @@ const Search = (props) => {
       inputSelector: '#search_input_react',
       algoliaOptions: algolia.algoliaOptions,
       autocompleteOptions: {
+        openOnFocus: true,
```

Comment:
As a bonus I set this option because it's quite useful from a UX point of view.
If a user closes the block with search suggestions (eg. blur event), then they click on the search input again, these results are immediately displayed, which is very convenient.